src/SonarScanner.MSBuild.PreProcessor/FilePermissionsWrapper.cs

```diff
@@ -33,35 +33,25 @@ public void Set(string destinationPath, int mode)
     {
         if (operatingSystemProvider.IsUnix())
         {
-            if (operatingSystemProvider.OperatingSystem() is PlatformOS.Alpine)
+            // https://github.com/Jackett/Jackett/blob/master/src/Jackett.Server/Services/FilePermissionService.cs#L27
+            var process = new Process
             {
-                // https://github.com/Jackett/Jackett/blob/master/src/Jackett.Server/Services/FilePermissionService.cs#L27
-                var process = new Process
+                StartInfo = new ProcessStartInfo
                 {
-                    StartInfo = new ProcessStartInfo
-                    {
-                        RedirectStandardError = true,
-                        UseShellExecute = false,
-                        CreateNoWindow = true,
-                        WindowStyle = ProcessWindowStyle.Hidden,
-                        FileName = "chmod",
-                        Arguments = $"""{Convert.ToString(mode, 8)} "{destinationPath}" """,
-                    }
-                };
-                process.Start();
-                var stdError = process.StandardError.ReadToEnd();
-                process.WaitForExit();
-                if (process.ExitCode != 0)
-                {
-                    throw new InvalidOperationException(stdError);
+                    RedirectStandardError = true,
+                    UseShellExecute = false,
+                    CreateNoWindow = true,
+                    WindowStyle = ProcessWindowStyle.Hidden,
+                    FileName = "chmod",
+                    Arguments = $"""{Convert.ToString(mode, 8)} "{destinationPath}" """,
```

Comment:
Do we need to take care of path normalization before we pass it to chmod? Until now we use `destinationPath` only to pass it to .Net APIs which should support `/` and `\` as directory separators. Maybe we should do:
```suggestion
                    Arguments = $"""{Convert.ToString(mode, 8)} "{Path.GetFullPath(destinationPath)}" """,
```
According to the [example on the help page](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.getfullpath?view=net-8.0), the method translates the directory separator.